tests/units/nulecule/test_nulecule.py

```diff
@@ -0,0 +1,97 @@
+import mock
+import unittest
+from atomicapp.nulecule.base import Nulecule
+
+
+class TestNuleculeRun(unittest.TestCase):
+    """Test Nulecule run"""
+
+    def test_run(self):
+        provider = 'docker'
+        dryrun = False
+        n = Nulecule('some-id', '0.0.2', {}, [], 'some/path')
+        mock_component_1 = mock.Mock()
+        mock_component_2 = mock.Mock()
+        n.components = [mock_component_1, mock_component_2]
+        n.run(provider)
+        mock_component_1.run.assert_called_once_with(provider, dryrun)
+        mock_component_2.run.assert_called_once_with(provider, dryrun)
+
+
+class TestNuleculeStop(unittest.TestCase):
+    """Test Nulecule stop"""
+
+    def test_stop(self):
+        provider = 'docker'
+        dryrun = False
+        n = Nulecule('some-id', '0.0.2', {}, [], 'some/path')
+        mock_component_1 = mock.Mock()
+        mock_component_2 = mock.Mock()
+        n.components = [mock_component_1, mock_component_2]
+        n.stop(provider)
+        mock_component_1.stop.assert_called_once_with(provider, dryrun)
+        mock_component_2.stop.assert_called_once_with(provider, dryrun)
+
+
+class TestNuleculeLoadConfig(unittest.TestCase):
+    """Test Nulecule load_config"""
+
+    def test_load_config(self):
+        config = {'group1': {'a': 'b'}}
+        n = Nulecule('some-id', '0.0.2', {}, [], 'some/path')
+        mock_component_1 = mock.Mock()
+        n.components = [mock_component_1]
+        mock_component_1.config = {
+            'group1': {'a': 'c', 'k': 'v'},
+            'group2': {'1': '2'}
+        }
+        n.load_config(config)
+        self.assertEqual(n.config, {
+            'group1': {'a': 'c', 'k': 'v'},
+            'group2': {'1': '2'}
+        })
```

Comment:
I'm confused about what this is testing. on line 40 we are setting `config = {'group1': {'a': 'b'}}` and then calling `n.load_config(config)` with that value but I don't see where **'a':'b'** make it into the configuration? I still think for each of these if we had a few sentences about the goal of the test it would be useful.